code/game/jobs/job/security_vr.dm

```diff
@@ -2,10 +2,14 @@
 	disallow_jobhop = TRUE
 
 	access = list(access_security, access_eva, access_sec_doors, access_brig, access_armory,
-			            access_forensics_lockers, access_morgue, access_maint_tunnels, access_all_personal_lockers,
-			            access_research, access_engine, access_mining, access_construction, access_mailsorting,
-			            access_heads, access_hos, access_RC_announce, access_keycard_auth, access_gateway, access_external_airlocks)
+						access_forensics_lockers, access_morgue, access_maint_tunnels, access_all_personal_lockers,
+						access_research, access_engine, access_mining, access_construction, access_mailsorting,
+						access_heads, access_hos, access_RC_announce, access_keycard_auth, access_gateway, access_external_airlocks)
 	minimal_access = list(access_security, access_eva, access_sec_doors, access_brig, access_armory,
-			            access_forensics_lockers, access_morgue, access_maint_tunnels, access_all_personal_lockers,
-			            access_research, access_engine, access_mining, access_construction, access_mailsorting,
-			            access_heads, access_hos, access_RC_announce, access_keycard_auth, access_gateway, access_external_airlocks)
\ No newline at end of file
+						access_forensics_lockers, access_morgue, access_maint_tunnels, access_all_personal_lockers,
+						access_research, access_engine, access_mining, access_construction, access_mailsorting,
+						access_heads, access_hos, access_RC_announce, access_keycard_auth, access_gateway, access_external_airlocks)
+
+/datum/job/officer
+	total_positions = 5
+	spawn_positions = 5
```

Comment:
We _really_ don't need more security officers. Warden, Detective, and HoS already bump these numbers.

Reply:
4/4. The only reason I'm suggesting further bump is because all the base department jobs in other departments (Scientist, Medical Doctor, Station Engineer) are 5/5 or 3/5

Reply:
Is MD at 5? I thought it was 4.

Reply:
Yes, MD and Engineer are 5/5

Reply:
Oh the hell with it.